DEVELOPMENT.md

```diff
@@ -9,21 +9,47 @@ The Pixie project currently consists of three main components:
 This document outlines the process for setting up the development environment for each of these components.
 
 ## Setting up the Environment
+There are two different ways to run a development environment. The first and easiest
+approach is via Docker. If you plan to use a non-Minikube environment (GKE, EKS, etc)
+for running Vizier/Pixie Cloud, use this first approach via Docker. The second approach
+is using Make and Minikube.
 
-To setup the developer environment required to start building Pixie's components, it is easiest to use our docker image which has the necessary tools and packages. If you plan to use a non-minikube environment (gke, eks, etc) for running Vizier/Pixie Cloud, this can be done by running our script:
+### Running Via Docker
+To setup the developer environment required to start building Pixie's components,
+it is easiest to run the `run_docker.sh` script. The following script will run the
+Docker container and dump you out inside the docker container console from which
+you can run all the necessary tools to build, test, and deploy Pixie.
```

Comment:
might want to say deploy Pixie in development mode here, just so readers don't think these steps are necessary to deploy "normal" Pixie